client/executor/runtime-test/src/lib.rs

```diff
@@ -286,9 +286,9 @@ sp_core::wasm_export_functions! {
 		run().is_some()
 	}
 
-	// Just some test to make sure that `sp-allocator` compiles on `no_std`.
-	fn test_sp_allocator_compiles() {
-		sp_allocator::FreeingBumpHeapAllocator::new(0);
+	// Just some test to make sure that `sc-allocator-primitives` compiles on `no_std`.
+	fn test_sc_allocator_primitive_compiles() {
```

Comment:
This makes no sense anymore. Please remove it.

Reply:
The crate itself doesn't need to be `no_std` anymore.

Reply:
Why did it need to be no_std in the first place? (Just asking in case I'm missing something)

Reply:
There was once the plan to use it from wasm, but this was thrown away. So, it is not needed anymore.

Reply:
ok crate is standardised.